### PR TITLE
Fix docs description format

### PR DESCRIPTION
**Pull request by ahitacat**
Some description lack of uppercase. Staleness used default instead of choices.

### CODE REVIEW

File: plugins/inventory/insights.py [hunk 1]
```diff
@@ -35,15 +36,15 @@
         type: str
       staleness:
         description: Choose what hosts to return, based on staleness
-        default: [ 'fresh', 'stale', 'unknown' ]
+        choices: [ 'fresh', 'stale', 'unknown' ]
```
Comment (ptoscano) [inline plugins/inventory/insights.py:39]: hm I don't think this is correct: `staleness` is actually a list, and the mentioned string is the default (i.e. 3 elements)

**Review by ptoscano — CHANGES_REQUESTED** [review]
mostly LGTM, only one mistake

**Comment by ahitacat** [comment]
I have updated the docs as well.

**Comment by ptoscano** [comment]
> I have updated the docs as well.

This is fine, although my plan/idea is to do a docs regeneration right before a release, to minimize the updates.

**Review by ptoscano — APPROVED** [review]

**Pull request merged**